pkgs/development/python-modules/git-revise/default.nix

```diff
@@ -1,30 +1,26 @@
 { lib
 , buildPythonPackage
-, fetchPypi
 , pythonOlder
-, tox
-, pytest
-, pylint
-, mypy
-, black
+, git
+, fetchFromGitHub
+, pytestCheckHook
 }:
 
 buildPythonPackage rec {
   pname = "git-revise";
-  version = "0.5.1";
+  version = "0.6.0";
 
-  src = fetchPypi {
-    inherit pname version;
-    sha256 = "0l3xhg00106p7ysg4gl9dna2zcrax58mzmm0ajjaxw58jfn8wsf1";
+  # Missing tests on PyPI
+  src = fetchFromGitHub {
+    owner = "mystor";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "03v791yhips9cxz9hr07rhsgxfhwyqq17rzi7ayjhwvy65s4hzs9";
   };
 
   disabled = pythonOlder "3.6";
 
-  checkInputs = [ tox pytest pylint mypy black ];
-
-  checkPhase = ''
-    tox
-  '';
+  checkInputs = [ git pytestCheckHook ];
```

Comment:
I presume this skips running the type-checking, linting and formatting checks from the [`tox.ini`](https://github.com/mystor/git-revise/blob/v0.6.0/tox.ini). Not sure if all of those are worth keeping (the mypy probably is at least), but felt worth noting.